pimcore/lib/Pimcore/Bundle/EcommerceFrameworkBundle/Resources/config/payment_manager.yml

```diff
@@ -31,6 +31,11 @@ services:
             $options: []
             $formFactory: '@form.factory'
 
+    Pimcore\Bundle\EcommerceFrameworkBundle\PaymentManager\Payment\;Mpay24Seamless:
```

Comment:
is this semicolon a mistake or on purpose? `;Mpay24Seamless`

Reply:
It's a mistake. It should look the same way as it is for other payment providers:

```   
 Pimcore\Bundle\EcommerceFrameworkBundle\PaymentManager\Payment\WirecardSeamless:
        arguments:
            $options: []
            $templatingEngine: '@templating.engine.delegating'
            $session: '@session'
```

Reply:
could you change it? :)